<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rdb_ycyx_dialog_base</class>
 <widget class="QDialog" name="Rdb_ycyx_dialog_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>402</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>402</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>相关显示设置</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>实时系统设置</string>
     </property>
     <widget class="QLabel" name="textLabel2_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>秒</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_pRefresh_checkbox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>190</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>是否定时刷新</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabel2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新周期：</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_pStation_checkbox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>190</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>显示按厂站排序</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="m_pTime_spinbox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>实时遥测设置</string>
     </property>
     <widget class="QCheckBox" name="m_pYcmanset_checkbox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>190</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>是否显示锁定标志</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_pYcxs_checkbox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>170</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>是否显示遥测系数</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_pYcky_checkbox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>170</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>是否显示遥测可疑</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_pYccal_checkbox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>180</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>是否显示计算标志</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string>实时电度/遥信设置</string>
     </property>
     <widget class="QCheckBox" name="m_pYxmanset_checkbox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>280</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>是否显示锁定标志</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_pKwhshow_checkbox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>300</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>电度显示方式(选中为显示表码)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_pYxcal_checkbox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>300</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>是否显示计算标志</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="m_pOk_button">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>258</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="m_pCancel_button">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
